testsuite/integration-arquillian/servers/app-server/jetty/81/src/test/java/org/keycloak/testsuite/arquillian/jetty/Jetty81AppServerTest.java

```diff
@@ -0,0 +1,5 @@
+package org.keycloak.testsuite.arquillian.jetty;
+
+public class Jetty81AppServerTest extends AbstractJettyAppServerTest {
```

Comment:
I get 

```
Tests in error: 
  Jetty81AppServerTest>AbstractJettyAppServerTest.testDeployingAndUndeploying:123 » NoClassDefFound
  Jetty81AppServerTest>AbstractJettyAppServerTest.testDeployingRESTApp:95 » NoClassDefFound
  Jetty81AppServerTest>AbstractJettyAppServerTest.testDeployingServletApp:67 » NoClassDefFound
  Jetty81AppServerTest>AbstractJettyAppServerTest.testServerStartupAndShutdown:43 » NoClassDefFound

...
java.lang.NoClassDefFoundError: org/eclipse/jetty/server/HttpConfiguration$Customizer
...
```

when I'm trying to build `integration-arquillian-servers-app-server`

Reply:
Yes, I accidentally broke it just before the push. 

Yesterday I spoke with @hmlnarik that we won't migrate 8.1 version to the new testsuite (due to [KEYCLOAK-8356](https://issues.jboss.org/browse/KEYCLOAK-8356)). I will remove this part shortly.